Simple Incremental/Assets/Scripts/Monobehaviours/PlayerMovementController.cs

```diff
@@ -15,43 +16,46 @@ public class PlayerMovementController : MonoBehaviour
 
     float horizontalForce;
     Rigidbody2D rigidBody;
+    SpriteRenderer spriteRenderer;
     Vector2 currentVelocity = Vector2.zero;
     float normalizeSpeed = 10f;  //Used to make velocity numbers look reasonable
     int groundLayerID;
     Animator anim;
-    int jumpHash = Animator.StringToHash("Jump");
     int groundedHash = Animator.StringToHash("Grounded");
 
     private void Awake()
     {
         rigidBody = GetComponent<Rigidbody2D>();
         anim = GetComponent<Animator>();
+        spriteRenderer = GetComponent<SpriteRenderer>();
         groundLayerID = LayerMask.NameToLayer("Ground");
     }
 
     private void Update()
     {
         horizontalForce = Input.GetAxisRaw(horizontalAxis);
 
-        if (Input.GetKeyDown("space") && anim.GetBool(groundedHash))
+        if (Input.GetKeyDown(KeyCode.Space) && anim.GetBool(groundedHash))
         {
-            anim.SetBool(jumpHash, true);
+            anim.SetTrigger("Jump");
+            rigidBody.AddForce(new Vector2(0f, jumpForce));
+        }
+        if (Input.GetKeyDown(KeyCode.LeftControl))
```

Comment:
Attacking is controlled elsewhere (Currently by the WeaponRangedController and the WeaponMeleeController).